docs/template_plugin/tests/functional/op_reference/logical_not.cpp

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
In #6867 logical.hpp is introduced to allow instantiation of binary logical operations. I think we can wait until this PR is merged and extend these member functions to include also \*LogicalNot\*. Similar as SLTs are implemented for logical operations [here](inference-engine/tests/functional/shared_test_classes/src/single_layer/logical.cpp).

Reply:
Done:)